configs/chicoma_cpu.json

```diff
@@ -1,20 +1,20 @@
 {
-    "cms_dir": "/users/moraru/Parsl-Project/exa-amd/cms_dir",
-    "work_dir": "/lustre/scratch5/moraru/work_dir_v1",
-    "cpu_account": "t25_ml-amd",
+    "cms_dir": "<path_to>/exa-amd/cms_dir",
+    "work_dir": "./work_dir_v1",
+    "cpu_account": "cpu_account",
     "elements": "Na-B-C",
     "formation_energy_threshold": -0.2,
     "num_workers": 128,
-    "initial_structures_dir":"/users/moraru/Parsl-Project/exa-amd/cms_dir/mpstrs",
+    "initial_structures_dir":"initial_structures",
```

Comment:
Same as above. Is it only the name of the directory or is it a path? If it is a path, change it to "./initial_structures" or "<path_to>/initial_structures".